src/main/scala/xiangshan/cache/dcache/loadpipe/LoadPipe.scala

```diff
@@ -131,9 +131,10 @@ class LoadPipe(id: Int)(implicit p: Parameters) extends DCacheModule with HasPer
   io.tag_read.valid := io.lsu.req.fire && !io.nack
 
   val s0_valid = io.lsu.req.fire
-  val s0_req = io.lsu.req.bits
+  val s0_req = WireInit(io.lsu.req.bits)
+  s0_req.vaddr := Mux(io.load128Req, Cat(io.lsu.req.bits.vaddr(io.lsu.req.bits.vaddr.getWidth - 1, 4), 0.U(4.W)), io.lsu.req.bits.vaddr)
   val s0_fire = s0_valid && s1_ready
-  val s0_vaddr = s0_req.vaddr
+  val s0_vaddr = Mux(io.load128Req, Cat(s0_req.vaddr(s0_req.vaddr.getWidth - 1, 4), 0.U(4.W)), s0_req.vaddr)
```

Comment:
Why can't it be reduced here to `val s0_vaddr=s0_req.vaddr`?

Reply:
The purpose here is to change the address to a 128-bit aligned address because I initiated a load128Req.